aesara/graph/fg.py

```diff
@@ -227,12 +227,12 @@ def disown(self) -> None:
         self.clients = {}
         self.apply_nodes = set()
         self.variables = set()
-        self.inputs = None
-        self.outputs = None
+        self.inputs = []
+        self.outputs = []
```

Comment:
They were initialized as `[]` in the `__init__` and this way we don't have to make them `Optional`.